docs/user/guides/connecting-git-account.rst

```diff
@@ -1,22 +1,70 @@
-How to connect your Read the Docs account to your Git repository
-================================================================
+How to connect your Read the Docs account to your Git provider
+==============================================================
+
+In this how-to,
+you are shown the steps to connect an account on GitHub, Bitbucket, or GitLab with your Read the Docs account.
```

Comment:
```suggestion
you are shown the steps to connect an account on |git_provider_and| with your Read the Docs account.
```

:D

Reply:
Exactly this! I've just merged the style guide changes to `main`, then to `diataxis/main` and then into this PR so I can use the substitutions here :)